src/resources/Catalogs/CatalogInterfaces.ts

```diff
@@ -33,7 +33,7 @@ export interface CreateCatalogConfigurationModel {
     /**
      * The unique identifier of the catalog configuration.
      */
-    id: string;
+    id?: string;
```

Comment:
Does it really need to be optional or can we use `New` instead ?

Reply:
True, I'll revert this change